build.gradle.kts

```diff
@@ -0,0 +1,559 @@
+import com.diffplug.gradle.spotless.SpotlessExtension
+import com.google.protobuf.gradle.*
+import com.jfrog.bintray.gradle.BintrayExtension
+import com.jfrog.bintray.gradle.BintrayExtension.*
+import groovy.lang.GroovyObject
+import io.morethan.jmhreport.gradle.JmhReportExtension
+import me.champeau.gradle.JMHPluginExtension
+import nebula.plugin.release.git.opinion.Strategies
+import net.ltgt.gradle.errorprone.ErrorProneOptions
+import net.ltgt.gradle.errorprone.ErrorPronePlugin
+import org.gradle.api.plugins.JavaPlugin.*
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.jfrog.gradle.plugin.artifactory.dsl.ArtifactoryPluginConvention
+import org.jfrog.gradle.plugin.artifactory.dsl.PublisherConfig
+import org.jfrog.gradle.plugin.artifactory.dsl.ResolverConfig
+import ru.vyarus.gradle.plugin.animalsniffer.AnimalSniffer
+import ru.vyarus.gradle.plugin.animalsniffer.AnimalSnifferPlugin
+
+plugins {
```

Comment:
would it make sense to have 2 `plugins` closures? One for the things we apply universally and one for the bits we don't?

Reply:
hm. looks like we can't have 2 of them, but maybe we can separate the `apply false` from the others, just for clarity?

Reply:
Good idea